public/app/features/alerting/unified/mocks/server/configure.ts

```diff
@@ -104,7 +104,7 @@ export function mimirDataSource() {
         manageAlerts: true,
       },
     },
-    { alerting: true }
+    { alerting: true, module: 'core:plugin/alertmanager' }
```

Comment:
Datasources have to be more accurate with the latest updates, so you need to specify a module so the `plugin loader` logic will actually "load" the datasource correctly in tests

Reply:
is this module correct?, should not be `core:plugin/prometheus`?

Reply:
same as before, what about  setting `module: core:plugin/prometheus` in the mockDataSource when using `DataSourceType.Prometheus` type?

Reply:
Yes you're right, it should have been `prometheus` here. I changed that, and there were some additional MSW handlers I had to add as well.

I'm not sure about tying the logic together based on different datasource types - can I think about it when I make this work alongside Konrad's work from the pagination PR?